jumpapp

```diff
@@ -164,9 +166,11 @@ print_windows() {
 }
 
 get_subsequent_window() {
-    if [[ -n "$in_reverse" ]]; then
+    if [[ -n $in_reverse ]]; then
         get_prev_window "$@"
```

Comment:
Will it be weird to have cycling forward through windows go in one order and cycling in reverse through windows go in a different order? Would it be possible to update the logic here so it does whatever `get_last_focused_window` does but in the other direction?